ps_contactinfo.tpl

```diff
@@ -52,9 +52,11 @@
       <br>
       {* First tag [1][/1] is for a HTML tag. *}
       {l
-        s='Email: %email%'
+        s='Email: [1]%email%[/1]'
         sprintf=[
-          '%email%' => $contact_infos.email
+          '%email%' => $contact_infos.email,
+          '[1]' => '<a href="mailto:'|qcat:$contact_infos.email|cat:'">',
```

Comment:
I've removed it because I think that this class is related to the classic theme (users should add their new style in the override)
I'm not pretty sure, can you check it with @nihco2 please?